e2e-tests/IIoTPlatform-E2E-Tests/Standalone/A_PublishSingleNodeStandaloneDirectMethodTestTheory.cs

```diff
@@ -192,6 +215,25 @@ IIoTMultipleNodesTestContext context
             // Wait some time to generate events to process.
             await Task.Delay(TestConstants.DefaultTimeoutInMilliseconds, cts.Token);
 
+            //Create request for GetConfiguredNodesOnEndpoint method call
+            var nodesOnEndpoint = new PublishedNodesEntryModel();
+            nodesOnEndpoint.EndpointUrl = request.EndpointUrl;
+            var requestGetConfiguredNodesOnEndpoint = nodesOnEndpoint.ToApiModel();
+
+            //Call GetConfiguredNodesOnEndpoint direct method
+            var responseGetConfiguredNodesOnEndpoint = await TestHelper.CallMethodAsync(_iotHubClient, _iotHubPublisherDeviceName, _iotHubPublisherModuleName, new MethodParameterModel {
+                Name = "GetConfiguredNodesOnEndpoint",
```

Comment:
I see both `GetConfiguredNodesOnEndpoint` and `GetConfiguredNodesOnEndpoint_V1` being used as direct method name. Do they both work ? Than should not be the case as far as I understand. Can you please check ? Also, let's have constants for direct method names and use them instead of defining them manually. Please add those constants for both this, `PublishNodes_V1` and any other direct method name that you find.

Reply:
Ah, ok, I see that one test is for `2.5` version of OPC Publisher and the other one is for `2.8` version. So we also have `PublishNodes` and `PublishNodes_V1`. Can you please create two sets of constants then ? One for legacy and one for current versions of OPC Publisher.

Reply:
added constants for dm names